pyproject.toml

```diff
@@ -44,15 +44,16 @@ dependencies = [
   "netCDF4",
   "numpy",
   "pandas",
-  "pysat >= 3.1",
+  "pysat >= 3.2",
   "requests",
+  "scipy >= 1.4",
   "xarray"
 ]
 
 [project.optional-dependencies]
 pysatcdf = ["pysatCDF"]
 test = [
-  "coveralls < 3.3",
+  "coveralls",
```

Comment:
So, the coveralls folks don't recommend installing coveralls with the test.  If we remove this, the coveralls-actions *might* work.

Reply:
Fixed with #241